volume/drivers/buse/buse.go

```diff
@@ -109,12 +115,13 @@ func Init(params map[string]string) (volume.VolumeDriver, error) {
 		dlog.Println("Could not enumerate Volumes, ", err)
 	}
 
+	inst.cl = &clusterListener{}
 	c, err := cluster.Inst()
 	if err != nil {
 		dlog.Println("BUSE initializing in single node mode")
 	} else {
 		dlog.Println("BUSE initializing in clustered mode")
-		c.AddEventListener(inst)
+		c.AddEventListener(inst.cl)
```

Comment:
Yes I was not sure why it was added in the first place. I can remove it completely.